Blockzilla/BrowserViewController.swift

```diff
@@ -1012,7 +1015,8 @@ extension BrowserViewController: HomeViewDelegate {
         let numberOfTrackersBlocked = getNumberOfLifetimeTrackersBlocked()
         let appStoreUrl = URL(string:String(format: "https://mzl.la/2GZBav0"))
         // Add space after shareTrackerStatsText to add URL in sentence
-        let text = String(format: UIConstants.strings.shareTrackerStatsText + " ", AppInfo.productName, String(numberOfTrackersBlocked))
+        let shareTrackerStatsText = "%@, the privacy browser from Mozilla, has already blocked %@ trackers for me. Fewer ads and trackers following me around means faster browsing! Get %@ for yourself here"
```

Comment:
do we have to change this one?

Reply:
Just updated this to now say "Get Focus for yourself here" instead of the entire app name. i.e. it's back to how it originally was before localization.